### DISCUSSION TITLE
Add declarationSortOrder option

### DISCUSSION THREAD

**Discussion by SvenBudak** [eslint/eslint#14746] [Ideas]
Hi, i am wondering a bit why there is no option like `declarationSortOrder`: https://eslint.org/docs/rules/sort-imports#options

```
{
    "sort-imports": ["error", {
        "ignoreCase": false,
        "ignoreDeclarationSort": false,
        "ignoreMemberSort": false,
        "memberSyntaxSortOrder": ["none", "all", "multiple", "single"],
        "allowSeparatedGroups": false
    }]
}
```

We can tell eslint to not ignore the declarationSort. But if i run `fix eslint` nothing happens. He just report, that the Order should be sorted by aplhabet. Is there a feature like this planned?

**Comment by nzakas** [comment]
No there isn’t. We are no longer updating rules that just enforce stylistic preferences.

**Reply by SvenBudak** [reply]
Ok thanks for the fast reply. Someone know a good plugin for this?
